src/server.rs

```diff
@@ -145,6 +146,33 @@ impl Server {
         ServerBuilder::default().build()
     }
 
+    /// # Errors
+    ///
+    /// Errors when the call to `UA_Server_updateCertificate`
+    /// doesn't return `ua::StatusCode::GOOD`
+    pub fn update_certificate(
+        &self,
+        old_certificate: &ua::ByteString,
+        new_certificate: &ua::ByteString,
+        new_private_key: &ua::ByteString,
+        close_sessions: bool,
+        close_secure_channels: bool,
+    ) -> Result<()> {
+        let status_code = ua::StatusCode::new(unsafe {
+            UA_Server_updateCertificate(
+                self.0.as_ptr().cast_mut(),
```

Comment:
A colleague is checking whether exchanging certificates at runtime is supported by OPC UA and / or how it is used.
Right now it seems that this function could also be marked as `UA_THREADSAFE`.

Reply:
Okay so he could not tell me. He said it's probably not important that we have this feature right now. It would be more important to be able to add a certificate on server configuration before it runs.